assets/scss/adopters_wall.scss

```diff
@@ -0,0 +1,54 @@
+
```

Comment:
The whole styling of this file needs addressing. All items have obsolete indents and `@keyframes` should start at line one.

In addition, the file is missing a newline at the end.